style.css

```diff
@@ -122,13 +114,430 @@ body {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: white;
+    padding: 10%;
+  }
+
+  #listsx {
+    width: 100%;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+  }
+
+  hr {
+    width: 80%;
+    margin: 10%;
+    position: center;
+  }
+
+  .face {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+  }
+
+  .tilted_square {
+    margin: 5%;
+    height: 1.5rem;
+    width: 1.5rem;
+    background-color: #ff6b00;
+    transform: rotate(45deg);
+    display: inline-block;
+  }
+
+  .titles-skill {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+  }
+
+  #text {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+
+    /* or 138% */
+
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+  }
+
+  .boxp {
+    margin: 5%;
+    height: 1.5rem;
+    width: 1.5rem;
+    background-color: #ff6b00;
+    display: inline-block;
+  }
+
+  .dot {
+    margin-right: 3%;
+    height: 1.5rem;
+    width: 1.5rem;
+    background-color: #ff6b00;
+    display: inline-block;
+    border-radius: 50%;
+  }
+
+  .contact {
+    display: flex;
+    flex-direction: column;
+    align-content: space-around;
+    align-items: center;
+    background-color: white;
+    padding: 0%;
+    margin: 0%;
+  }
+
+  #name {
+    width: 80%;
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    padding: 5%;
+  }
+
+  #email {
+    width: 80%;
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    padding: 5%;
+    margin-bottom: 10%;
+  }
+
+  #para {
+    width: 80%;
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    border-bottom: none;
+    padding: 10%;
+    font-style: italic;
+    background: #fbf8f7;
+    margin-bottom: 10%;
+  }
+
+  #button1 {
+    display: flex;
+    flex-direction: row;
+    padding: 12px;
+    gap: 10px;
+    border: none;
+    background: #ff6b00;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+  }
+
+  .btn-3 {
+    background-color: white;
+    border: none;
+    outline: none;
+  }
+
+  /* project sample design */
+  .ex1 {
+    background:
+      linear-gradient(
+        180.45deg,
+        rgba(38, 38, 38, 0.103) 0.75%,
+        rgba(38, 38, 38, 0.9) 61.94%
+      ),
+      url("images/sada.png");
+    background-size: cover;
+    background-position: center;
+    padding: 21% 3%;
+    position: relative;
+    margin-top: 10%;
+    margin-bottom: 8%;
+    height: 386px;
+    width: 327px;
+  }
```

Comment:
- According to the project requirements, the content for each project should be aligned with the `flex` display property, please fix this 🙏 
- Your project looks good 👍, but for each of your project cards, the margin between the list of technology and the `See Project` button is not as it is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏 Please fix this 🙏 
Your project:
- Please note the marked section
![efweefef](https://github.com/Rdzaman20000/Portfolio/assets/73607512/7d4d61db-7bc6-40ad-a035-32b5b5c2ab88)
Figma template:
![efwefqe](https://github.com/Rdzaman20000/Portfolio/assets/73607512/6e1ab7aa-a128-42bc-a278-6d74f89c65ab)